### PR TITLE
Fix AIR related specs that were missed in #677

### PR DESCRIPTION
**Pull request by pasitlaothamatas**
### Summary

- Noticed there were some AIR related specs that still weren't fixed in #677 

### Checklist
*Please change `[ ]` to `[x]` when you are ready.*
- [ ] I have read and followed the code conventions detailed in [README.md](../README.md) (*This will save time for both you and the reviewer!*).
- [ ] If this pull request is longer then **500** lines, I have provided *explicit* justification in the summary above explaining why I *cannot* break this up into multiple pull requests (*Small PR's are faster and less painful for everyone involved!*).

### CODE REVIEW

File: specs.md [hunk 1]
```diff
@@ -68,13 +68,13 @@ FSM-18 | Brake actuation reporting | - The FSM must report the brake actuation O
 ID | Title | Description | Associated Competition Rule(s)
 --- | --- | --- | ---
 FSM-17 | Default state | The FSM state machine must begin in the AIR-Open state by default.
-FSM-12 | Exiting the AIR-Open state | The FSM must enter the AIR-Closed state when the BMS closes the AIR+ and AIR-.
+FSM-12 | Exiting the AIR-Open state | The FSM must enter the AIR-Closed state when the BMS closes the AIR+.
```
Comment (pasitlaothamatas) [inline specs.md:71]: @kirons96 Should I also add "or when the BMS detects AIR+ and AIR- are closed" or does this suffice

Comment (kirons96) [reply specs.md:71]: AIR+ alone should be fine as AIR- is less meaningful (i.e. during precharge)

**Review by kirons96 — APPROVED** [review]

File: specs.md (hunk 1 above)
Comment (pasitlaothamatas) [reply specs.md:71]: @kirons96 Actually before I merge this in could you explain why AIR- is less meaningful

Comment (kirons96) [reply specs.md:71]: AIR- closes during precharge
AIR+ is open during that time
we don't want to be in the "Air-Closed state" during precharge, only after it

**Pull request merged**